sparse/src/KokkosSparse_LUPrec.hpp

```diff
@@ -45,8 +46,9 @@ class LUPrec : public KokkosSparse::Experimental::Preconditioner<CRS> {
   using ScalarType = typename std::remove_const<typename CRS::value_type>::type;
   using EXSP       = typename CRS::execution_space;
   using MEMSP      = typename CRS::memory_space;
+  using DEVC       = typename Kokkos::Device<EXSP, MEMSP>;
```

Comment:
Minor suggestion: can we just use "DEVICE" instead of "DEVC"?

Reply:
Done